extensions/binding/org.eclipse.smarthome.binding.bluetooth/src/main/java/org/eclipse/smarthome/binding/bluetooth/BeaconBluetoothHandler.java

```diff
@@ -47,7 +48,7 @@ public BeaconBluetoothHandler(@NonNull Thing thing) {
     @Override
     public void initialize() {
         try {
-            address = new BluetoothAddress(getConfig().get(BluetoothBindingConstants.CONFIGURATION_ADDRESS).toString());
```

Comment:
Please revert this change - the address is defines as a configuration parameter, so it must exist for this handler.
We rather need to figure out why you have the situation that you have Things that only have the address set as a property, but not as a configuration parameter.

Reply:
OK, I will try to see why my Thing doesn't have the address in it's configuration.